add-on/panels/js/panel.jsx

```diff
@@ -20,19 +20,14 @@ loop.panel = (function(_, mozL10n) {
     handleButtonClick: function() {
       loop.requestMulti(
         ["OpenGettingStartedTour", "getting-started"],
-        ["SetLoopPref", "gettingStarted.latestFTUVersion", FTU_VERSION],
-        ["SetPanelHeight"]).then(function() {
+        ["SetLoopPref", "gettingStarted.latestFTUVersion", FTU_VERSION]
+      ).then(function() {
           var event = new CustomEvent("GettingStartedSeen");
           window.dispatchEvent(event);
         }.bind(this));
       this.closeWindow();
     },
 
-    componentWillMount: function() {
```

Comment:
Please test to be sure that the window is still sized correctly in a new profile.